integrations/opensearch/src/haystack_integrations/document_stores/opensearch/document_store.py

```diff
@@ -45,7 +46,10 @@ def __init__(
         mappings: Optional[Dict[str, Any]] = None,
         settings: Optional[Dict[str, Any]] = DEFAULT_SETTINGS,
         create_index: bool = True,
-        http_auth: Any = None,
+        http_auth: Any = (
+            Secret.from_env_var("OPENSEARCH_USERNAME", strict=False),  # noqa: B008
+            Secret.from_env_var("OPENSEARCH_PASSWORD", strict=False),  # noqa: B008
+        ),
```

Comment:
Let's please make this List instead of Tuple for yaml serializability:
```suggestion
        http_auth: Any = [
            Secret.from_env_var("OPENSEARCH_USERNAME", strict=False),  # noqa: B008
            Secret.from_env_var("OPENSEARCH_PASSWORD", strict=False),  # noqa: B008
        ],
```